functions-and-operators/locking-functions.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Locking Functions
+summary: Learn about user-level locking functions in TiDB.
+---
+
+# Locking Functions
+
+TiDB supports most of the user-level [locking functions](https://dev.mysql.com/doc/refman/5.7/en/locking-functions.html) available in MySQL 5.7.
+
+## Supported functions
+
+| Name                                                                                                                 | Description                                       |
+|:---------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
+| [`GET_LOCK(lockName, timeout)`](https://dev.mysql.com/doc/refman/5.7/en/locking-functions.html#function_get-lock)    | Acquire an advisory lock                          |
```

Comment:
should we add the restriction information of the lock name in this doc?
like: the length of lock name should <= 64 characters

Reply:
I don't think it's necessary. 64 chars is the same limit in mysql for lock name, and the same limit for column / table names.

Reply:
I think we should document this <=64 chars requirement for people that don't have a MySQL background. Or do we expect them to check the docs on mysql.com?